### PR TITLE
IAR-513: Spatial Anchors

### PR DESCRIPTION
**Pull request by holo-istemi**

### CODE REVIEW

File: BUILD.gn [hunk 1]
```diff
@@ -1,667 +1,667 @@
...
+    visibility = [ "//:default" ]
+
+    sources = []
+    complete_static_lib = true
+    suppressed_configs += [ "//build/config/compiler:thin_archive" ]
+    defines = []
+
+    deps = [
+      ":webrtc_common",
+"//remoting:remoting_all","//third_party/idl:idl","//remoting:remoting_all","//third_party/idl:idl",      "//remoting:remoting_all",
```
Comment (Holo-FloWandinger) [inline BUILD.gn]: Nothing special here. But please preserve the code layout :)

File: .vscode/tasks.json [hunk 2]
```diff
@@ -1329,6 +1338,23 @@
...
+        },
+        "command": "flatc",
+        "args": [
+          "--cpp",
+          "--scoped-enums",
+          // "--filename-suffix", "",
+          "--cpp-str-flex-ctor",
+          "-o",
+          "isar.protocol/",
+          "anchoring.fbs"
```
Comment (aidanhowie19) [inline .vscode/tasks.json:1329]: Multiple flatbuffers files can be generated with a single task. See https://github.com/Holo-Light/webrtc/commit/0ffe9dc36796172efc094327bcca3e3f48c9698a for how to do it.
Note: you do not need to pass the -I option, this is for includes which is no longer relevant

**Review by aidanhowie19 — CHANGES_REQUESTED** [review]
The build.gn file has not been manually changed, it has been done due to the build tasks which we use. Can we please revert the commit which changed this file

File: .vscode/tasks.json (hunk 2 above)
Comment (holo-istemi) [reply .vscode/tasks.json:1329]: Fixed: 5f7bfb74e87a4e08e0a3be3e6f7b0f7bb64388d5

File: BUILD.gn (hunk 1 above)
Comment (holo-istemi) [reply BUILD.gn]: Fixed 8e83c4a88f7c5ed337a5fd16930e677d0ed79367

**Review by aidanhowie19 — APPROVED** [review]
All my points have now been resolved

**Pull request merged**